learn/update_and_migration/migrating_cloud.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Migrating to Meilisearch Cloud — Meilisearch Documentation
+description: Meilisearch Cloud is the recommended way of using Meilisearch. This guide walks you through migrating Meilisearch from a self-hosted installation to Meilisearch Cloud.
+---
+
+# Migrating to Meilisearch Cloud
+
+Meilisearch Cloud is the recommended way of using Meilisearch. This guide walks you through migrating Meilisearch from a self-hosted installation to Meilisearch Cloud.
+
+## Requirements
+
+To follow this guide you need:
+
+- A running Meilisearch instance
+- A command-line terminal
+- A Meilisearch Cloud account
+
+## Export a dump from your self-hosted installation
+
+To migrate Meilisearch, you must first [export a dump](/learn/advanced/dumps). A dump is a compressed file containing all your indexes, documents, and settings.
+
+To export a dump, make sure your self-hosted Meilisearch instance is running. Then, open your terminal and run the following command, replacing `MEILISEARCH_URL` with your instance's address:
+
+```sh
+curl  -X POST 'http://MEILISEARCH_URL:7700/dumps'
+```
+
+Meilisearch will return a summarized task object and begin creating the dump. [Use the returned object's `taskUid` to monitor its progress.](/learn/async/asynchronous_operations)
+
+Once the task has been completed, you can find the dump in your project's dump directory. By default, this is `/dumps`.
+
+<Capsule intent="note" title="Command-line options and environment variables">
+Instance configuration options and experimental features that can only be activated at launch are not included in dumps.
+
+Once you have successfully migrated your data to Meilisearch Cloud, use the project overview interface to reactivate available options. Not all instance options are supported in the Cloud.
+</Capsule>
+
+## Create a Meilisearch Cloud project and import dump
+
+Navigate to Meilisearch Cloud in your browser and log in. If you don't have a Meilisearch Cloud account yet, [create one for free](https://cloud.meilisearch.com/register).
+
+You can only import dumps into new Meilisearch Cloud projects. Create a new project by clicking on the "New project" button:
+
+![The Meilisearch Cloud menu, featuring the "New Project" button](https://raw.githubusercontent.com/meilisearch/documentation/migration-self-hosted-to-cloud/assets/images/cloud-migration/1-new-project.png)
```

Comment:
I think the position will change soon, we might want to make sure it's communicated to the docs team in order to update screenshots 🤔

Reply:
good to know, thanks for the heads-up!

Reply:
Hey @guimachiavelli, you can check the new designs on cloud.meilisearch.dev, it will be released in production today.
**With free trial:**
![Screenshot 2023-10-26 at 11 04 08](https://github.com/meilisearch/documentation/assets/44903069/0f163264-9190-4dcb-985f-a8f2843ec37d)
![Screenshot 2023-10-26 at 11 03 54](https://github.com/meilisearch/documentation/assets/44903069/7020c9ec-b77d-446d-aad3-a49d5b93f6ec)

**Without free trial:**
![Screenshot 2023-10-26 at 11 04 31](https://github.com/meilisearch/documentation/assets/44903069/09cdcc77-a60d-46fa-b4e5-0f712e736464)
![Screenshot 2023-10-26 at 11 04 20](https://github.com/meilisearch/documentation/assets/44903069/7fdc7773-5c64-47c9-b0b3-ab59ce072001)